### ISSUE TITLE
RSS torrent update is bailing because of TPE name resolution

### ISSUE THREAD

**Issue by weidercs** [evilhero/mylar#1548]
Looks like it's not getting to checking the other torrent providers (WWT in particular) because of an error with TPE...

```
25-Jan-2017 09:36:04 - INFO    :: CP Server Thread-9 : Attempting to run RSS Check Forcibly
25-Jan-2017 09:36:04 - INFO    :: Thread-13 : RSS Feed Check was last run at : 2017-01-25 09:33:37
25-Jan-2017 09:36:04 - INFO    :: Thread-13 : RSS Feed Check First Ever Run.
25-Jan-2017 09:36:04 - DEBUG   :: Thread-13 : Updating RSS Run time to : 2017-01-25 09:36:04
25-Jan-2017 09:36:04 - INFO    :: Thread-13 : [RSS] Initiating Torrent RSS Check.
25-Jan-2017 09:36:04 - INFO    :: Thread-13 : [RSS] Initiating Torrent RSS Feed Check on TorrentProject.
25-Jan-2017 09:36:14 - WARNING :: Thread-13 : Error fetching RSS Feed Data from DEM: ('Connection aborted.', gaierror(-3, 'Temporary failure in name resolution'))
25-Jan-2017 09:36:14 - INFO    :: Thread-13 : [RSS] Initiating RSS Feed Check for NZB Providers.
25-Jan-2017 09:36:14 - DEBUG   :: Thread-13 : [RSS] You have enabled 2 NZB RSS search providers.
25-Jan-2017 09:36:14 - DEBUG   :: Thread-13 : [RSS] Fetching items from experimental
25-Jan-2017 09:36:14 - DEBUG   :: Thread-13 : [RSS] Fetching items from nzb.su
25-Jan-2017 09:36:15 - DEBUG   :: Thread-13 : [RSS] (experimental) now being updated...
25-Jan-2017 09:36:15 - INFO    :: Thread-13 : [RSS] (experimental) 0 entries indexed.
25-Jan-2017 09:36:15 - DEBUG   :: Thread-13 : [RSS] (nzb.su) now being updated...
25-Jan-2017 09:36:15 - INFO    :: Thread-13 : [RSS] (nzb.su) 100 entries indexed.
25-Jan-2017 09:36:15 - INFO    :: Thread-13 : [RSS] 100 entries have been indexed and are now going to be stored for caching.
25-Jan-2017 09:36:21 - DEBUG   :: Thread-13 : Completed adding new data to RSS DB. Next add in 20 minutes
25-Jan-2017 09:36:21 - INFO    :: Thread-13 : [RSS] RSS Feed Check/Update Complete
```

**Comment by weidercs** [comment]
Quick update, looks like it's www.dnoid.me that isn't resolving...

**Comment by evilhero** [comment]
Yep, it looks like they're down for maintenance - so the unresolved error is killing things. Let me get a fix up for it..hang tight

**Comment by weidercs** [comment]
Don't they know comics come out on Wednesdays! :)  Thanks for all the work you do!

**Comment by evilhero** [comment]
heh - maybe they're testing non-load days ;)

All good and fixed in dev a9b7da8 : 
```
25-Jan-2017 13:20:52 - INFO    :: Thread-12 : [RSS] Initiating Torrent RSS Feed Check on TorrentProject.
25-Jan-2017 13:20:53 - WARNING :: Thread-12 : Error fetching RSS Feed Data from DEM: ('Connection aborted.', error(101, 'Network is unreachable'))
25-Jan-2017 13:20:53 - INFO    :: Thread-12 : [WWT] there were 50 results from the New Releases RSS Feed from WorldWideTorrents
25-Jan-2017 13:20:54 - DEBUG   :: Thread-12 : Completed adding new data to RSS DB. Next add in 20 minutes
```

**Comment by weidercs** [comment]
Fixed, here come the comics!!!  Thanks, closing
